pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -557,6 +557,23 @@ public static class Minion {
     public static final String CONFIG_OF_MINION_QUERY_REWRITER_CLASS_NAMES = "pinot.minion.query.rewriter.class.names";
   }
 
+  public static class ControllerJob {
+    /**
+     * Controller job ZK props
+     */
+    public static final String CONTROLLER_JOB_TYPE = "jobType";
+    public static final String CONTROLLER_JOB_TABLE_NAME_WITH_TYPE = "tableName";
+    public static final String CONTROLLER_JOB_ID = "jobId";
+    public static final String CONTROLLER_JOB_SUBMISSION_TIME = "submissionTime";
+    public static final String CONTROLLER_JOB_MESSAGES_COUNT = "messageCount";
```

Comment:
(minor) Keep the key and value consistent
```suggestion
    public static final String MESSAGE_COUNT = "messageCount";
```

Reply:
Ack